net-misc/ncman/ncman-0.0.1_p20190327-r2.ebuild

```diff
@@ -14,19 +17,15 @@ else
 	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
 fi
 
-DESCRIPTION="An ncurses UI for connman, forked from connman-json-client"
-HOMEPAGE="https://github.com/l4rzy/ncman"
-
 LICENSE="GPL-2"
```

Comment:
should be `GPL-2+`, see: https://github.com/l4rzy/ncman/blob/master/src/main.c#L7-L9